Specs/Scene/SkyBoxSpec.js

```diff
@@ -101,21 +89,15 @@ defineSuite([
             }
         });
 
-        var us = context.uniformState;
-        var frameState = createFrameState(context, createCamera({
-            offset : new Cartesian3(7000000.0, 0.0, 0.0),
-            near : 1.0,
-            far : 20000000.0
-        }));
-        frameState.mode = SceneMode.SCENE2D;
-        us.update(frameState);
-
-        var command = s.update(frameState);
-        expect(command).not.toBeDefined();
+        scene.mode = SceneMode.SCENE2D;
+        expect(scene.renderForSpecs()).toEqual([0, 0, 0, 255]);
+
+        scene.skyBox = skyBox;
+        expect(scene.renderForSpecs()).toEqual([0, 0, 0, 255]);
     });
 
     it('does not render without a render pass', function() {
```

Comment:
This test is similar to the sun one so `scene.frameState.passes.render = false` won't apply. You can update this test like the sun test.